docs/guides/so-you're-going-on-vacation.md

```diff
@@ -0,0 +1,39 @@
+# Vacation Planning
+
+So you're going on vacation, woohoo! <span aria-label="celebrate">🥳</span> Here
+are some things you should do before heading out and enjoying some well-earned
+time off.
+
+- [ ] Enter your time in [vPlanner](https://ibm.biz/BdZRkr)
+
+After logging in, you should head on over to the **Carbon Design and
+Development** team, find the relevant dates, and fill them out.
+
+- [ ] _(Optional)_ Setup your email auto-response
+
+You can do this in your email client by heading to the top-right hand side and
+selecting your profile image, then going to **Mail and calendar settings**.
+
+In this window, you can go to **Out of Office** and set the relevant dates and
+message you would like to have sent out when folks email you.
+
+- [ ] Set your Slack status
+
+Slack has a setting for setting your status for a pre-determined about of time,
```

Comment:
should this say:
> pre-determined _amount_ of time

Reply:
👍 want to use the suggestion syntax and I can commit with your changes? 👀